frontend/public/components/operator-lifecycle-manager/install-plan.tsx

```diff
@@ -34,7 +34,13 @@ export const InstallPlanRow: React.SFC<InstallPlanRowProps> = (props) => {
       <ResourceLink kind="Namespace" name={props.obj.metadata.namespace} title={props.obj.metadata.namespace} displayName={props.obj.metadata.namespace} />
     </div>
     <div className="hidden-xs col-sm-4 col-md-3 col-lg-2">
-      {props.obj.spec.clusterServiceVersionNames.map((csvName, i) => <span key={i}><ResourceIcon kind={referenceForModel(ClusterServiceVersionModel)} /> {csvName}</span>)}
+      {props.obj.spec.clusterServiceVersionNames.map((csvName, i) => <span key={i}>
+        { props.obj.status === null
+          ? <React.Fragment><ResourceIcon kind={referenceForModel(ClusterServiceVersionModel)} />{csvName}</React.Fragment>
+          : props.obj.status.phase === 'Complete'
+            ? <ResourceLink kind={referenceForModel(ClusterServiceVersionModel)} name={csvName} namespace={props.obj.metadata.namespace} title={csvName} />
+            : <React.Fragment><ResourceIcon kind={referenceForModel(ClusterServiceVersionModel)} />{csvName}</React.Fragment> }
```

Comment:
You should be able to avoid the nested ternary here:

```jsx
{_.get(props, 'obj.status.phase') === 'Complete'
  ? <ResourceLink kind={referenceForModel(ClusterServiceVersionModel)} name={csvName} namespace={props.obj.metadata.namespace} title={csvName} />
  : <React.Fragment><ResourceIcon kind={referenceForModel(ClusterServiceVersionModel)} />{csvName}</React.Fragment> }
```

Reply:
Thank you so much, that's what I'm looking for